routes/api.php

```diff
@@ -14,8 +13,9 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::post('register', 'AuthController@register')->name('register');
+Route::post('login', 'AuthController@login')->name('login');
+Route::post('logout', 'AuthController@logout')->name('logout')->middleware('auth:sanctum');
+Route::get('user', 'AuthController@user')->name('getUser')->middleware('auth:sanctum');
 
```

Comment:
It would be great if you could add the routes functionality and parameters to the README file

Reply:
It will be better id we can use [laravel-apidoc-generator](https://beyondco.de/docs/laravel-apidoc-generator/getting-started/installation)

Reply:
I just want the frontend team to be well informed about the endpoints they're consuming so if that works for them I don't mind. You can discuss this on the channel and let me know what was decided.

Reply:
resolved [here](https://github.com/BuildForSDGCohort2/Team-248-Backend/pull/29/commits/af155184041cc39acd7da999f7ef5caaff672884)